staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -250,3 +252,13 @@ type Zones interface {
 type PVLabeler interface {
 	GetLabelsForVolume(ctx context.Context, pv *v1.PersistentVolume) (map[string]string, error)
 }
+
+// InstanceMetadata is a collection of instance's type and nodeAddressed.
```

Comment:
Let's not mention specific fields explicitly here - adding a new field will make this comment outdated.
Maybe "InstanceMetadata contains metadata about the specific instance".

Reply:
Done, it's good.